2.4/root/usr/share/container-scripts/mongodb/mongodb.conf.template

```diff
@@ -1,7 +1,6 @@
 # mongodb.conf
 
-port = 27017
-pidfilepath = ${MONGODB_PID_FILE}
```

Comment:
Why are we removing MONGODB_PID_FILE? If we are to remove it, we should update the README / docs too (there might be OpenShift docs referring to it).